src/lwc/signals/core.ts

```diff
@@ -264,17 +301,18 @@ function $signal<T>(
           setter(newValue);
         }
       },
+      brand: Symbol.for("lwc-signals"),
```

Comment:
is this to identify the returned value as a signal?

Reply:
yeah